### PR TITLE
core database: batch_upsert validation + save a bunch of clones

### PR DESCRIPTION
**Pull request by spolu**
- Change of `DatabaseRow::new` signature saves a lot of clones
- Validate rows before inserting them in database (by generating their schema, which will be used later to update the schema in base)

### CODE REVIEW

File: core/src/databases/table_schema.rs [hunk 1]
```diff
@@ -185,21 +195,21 @@ mod tests {
             "field2": 1.2,
             "field3": "text",
             "field4": true,
-            "field6": ["array", "elements"],
-            "field7": {"key": "value"}
+            // "field6": ["array", "elements"],
```
Comment (fontanierh) [inline core/src/databases/table_schema.rs:198]: let's rm

File: core/src/stores/postgres.rs [hunk 2]
```diff
@@ -2390,12 +2390,19 @@ impl Store for PostgresStore {
             )
             .await?;
 
-        for (row_id, content) in contents {
-            let row_created = utils::now();
-            let row_data = content.to_string();
+        for row in rows {
```
Comment (fontanierh) [inline core/src/stores/postgres.rs:2393]: while we're here, can we iter collect that  ?

**Review by fontanierh — DISMISSED** [review]

File: core/src/stores/postgres.rs (hunk 2 above)
Comment (spolu) [reply core/src/stores/postgres.rs:2393]: won't change much as it would be an iter.forEach()

**Comment by spolu** [comment]
Added tests

**Pull request merged**